FFmpeg/WriteFFmpeg.cpp

```diff
@@ -3221,8 +3235,7 @@ WriteFFmpegPlugin::writeAudio(AVFormatContext* avFormatContext,
     MyAVFrame avFrame;
     const int audioReaderResult = audioReader_->read(avFrame);
     if ( audioReaderResult > 0 ) { //read successful
-        AVPacket pkt = {nullptr}; // data and size must be 0
-        av_init_packet(&pkt);
+        AVPacket *pkt = av_packet_alloc();
```

Comment:
I much prefer RIIA? Why not put allocation in the MyAVPacket constructor, and deallocation in its destructor?

Reply:
Done and added member access for the `AVPacket* _pkt` field.